hash/apple2_flop_clcracked.xml

```diff
@@ -38255,7 +39280,9 @@ license:CC0-1.0
 		<description>Library Magic (revision 0) (4am crack)</description>
 		<year>1987</year>
 		<publisher>McCarthy-McCormack</publisher>
-		<info name="programmer" value="Jane McCarthy and Kay McCormark, PhD." />
+		<info name="programmer" value="Jane McCarthy" />
+		<info name="programmer" value="Kay McCormark" />
+		<info name="programmer" value="PhD." />
```

Comment:
Oh no, another one snuck through.  But this time, it got split where it shouldn’t